prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -449,8 +455,13 @@ private boolean checkEntityInstanceAvailability(String entityName, String cluste
             if (entityType.equalsIgnoreCase(EntityType.PROCESS.toString())){
                 LOG.trace("Checking instance availability status for entity:{}, cluster:{}, "
                         + "instanceTime:{}", entity.getName(), clusterName, nominalTime, entityType);
-                AbstractWorkflowEngine wfEngine = WorkflowEngineFactory.getWorkflowEngine();
 
+                if ((System.currentTimeMillis() - nominalTime.getTime())/(1000*60*60*24) >= Integer.parseInt(
+                    RuntimeProperties.get().getProperty("worklflow.expiration.period", "7"))) {
```

Comment:
Make the property name to a constant field. Also, the property name can be : `workflow.history.expiration.period.days` (more intuitive and also indicates the time unit)